src/odemis/dataio/test/tiff_test.py

```diff
@@ -2729,6 +2732,77 @@ def test_convert_openfibsem_to_odemis_metadata(self):
             else:
                 self.assertEqual(value, data.content[0].metadata[key])
 
+PYRAMID_FILENAME = "test-pyramid.ome.tiff"
+class TestTileRead(unittest.TestCase):
+
+    def tearDown(self):
+        # clean up
+        try:
+            os.remove(PYRAMID_FILENAME)
+        except Exception:
+            pass
+
+    def test_tiled_read_and_projection(self):
+        """Test the tiled read functions"""
+
+        # storing full metadata for completeness
+        md = {
+            model.MD_PIXEL_SIZE: (7.738095238095199e-08, 7.738095238095199e-08),
+            model.MD_POS: (0.051781573195293, -0.007091270569792),
+            model.MD_ACQ_DATE: 1727296375,
+            model.MD_ROTATION: 6.1837053071795856,
+            model.MD_SHEAR: -4.3021142204224836e-16,
+            model.MD_DIMS: "YX",
+        }
+
+        # create a dummy overview pyramid with the same problematic shape
+        das = model.DataArray(
+            numpy.zeros((8194, 7143), dtype=numpy.uint16), metadata=md
+        )
+
+        # export the dummy overview pyramid
+        export(PYRAMID_FILENAME, das, pyramid=True)
+
+        # open the pyramid and create a projection
+        data = open_acquisition(PYRAMID_FILENAME)
+        stream = data_to_static_streams(data)[0]
+        proj = RGBSpatialProjection(stream)
+
+        # simulate 'moving' the stream
+        initial_rect = proj.rect.value
+        move = (25e-3, 10e-3)
+        stream.raw[0].metadata[model.MD_POS_COR] = move
+        proj.force_image_update()
```

Comment:
Is there no "subscribe" like mechanism on this property that automatically triggers `force_image_update`? This seems like a manual trigger is needed. Can that lead to unexpected behavior when people forget to call it?

Reply:
That's deliberate, the only way to move the image is modify the metadata which isn't a 'public' api so force the responsibility on the caller force-refresh

Reply:
Can be resolved then